### ISSUE TITLE
URL consistency for license badges

### ISSUE THREAD

**Issue by paulmelnikow** [badges/shields#4090]
I noticed today that the GitHub license badge uses `/license` whereas most of our other license badges use `/l`. I'd suggest we change GitHub to match the others and add a redirect.

The other one that uses `/license` is AUR; we could update that one too.

**Comment by calebcartwright** [comment]
Agreed with the objective of making them consistent, though as food for thought, weren't we elsewhere trying to use more explicit route naming conventions vs. shorthand? 

I think consistency across badge types would be great too, so that users don't have to remember that  one badge type uses a shorthand while another uses a more verbose/complete name (i.e. `l` for licenses but `coverage` for coverage, etc.)

**Comment by paulmelnikow** [comment]
That's a reasonable point. I'd be fine with switching over to `/license` which is definitely more readable… though that would also be a bit more work 😁

**Comment by paulmelnikow** [comment]
The other areas where we're using short identifiers pretty consistently are the version badges and the download badges. I'd be slightly less inclined to change those to longer forms, though really wouldn't mind doing it for `/l` to `/license`, I guess mostly because the `/l` badges seem a bit harder to visually recognize.

**Comment by PyvesB** [comment]
In my opinion, we should stick with `l`.

We've got over 15 badges using the short form and 2 using the long form. It would be quite a lot of work to migrate the majority over to the long form. I'm guessing we would have to either add redirectors or regexes in the path, with additional tests, adding complexity and maintenance overhead. Even though the performance hit would probably be very small, it will definitely not help with response times.

License is one of our core categories, keeping the short form is consistent with our other popular version and downloads categories. Yes, short forms are admittedly slightly harder to recognize visually, but people are probably used to it by now and it does save 6 characters on every single license URL. Very few services or "things" start with the letter `l`, so that reduces risk of confusion as well.

**Comment by calebcartwright** [comment]
> though that would also be a bit more work 😁

> We've got over 15 badges using the short form and 2 using the long form. It would be quite a lot of work to migrate the majority over to the long form.

That's a strong enough case on its own for me to vote for sticking with `l` too 👍
